src/main/java/su/litvak/chromecast/api/v2/Channel.java

```diff
@@ -153,7 +153,7 @@ public void run() {
                                 if (rp != null) {
                                     rp.put(jsonMSG);
                                 } else {
-                                    if (requestId == 0) {
+                                    if (requestId == 0 && parsed.has("responseType")) {
```

Comment:
I think this is incorrect to handle it this way. Instead of swallowing we should fire an `UNKNOWN` spontaneous event with JSON node. See `deliverEvent` method in the `EventListenerHolder` class.